plugins/modules/gitlab_project.py

```diff
@@ -172,6 +172,39 @@
       - This option is only used on creation, not for updates. This is also only used if I(initialize_with_readme=true).
     type: str
     version_added: "4.2.0"
+  builds:
+    description:
+      - C(private) Repository CI/CD allowed only to project members.
+      - C(disabled) Repository CI/CD is disabled.
+      - C(enabled) Repository CI/CD is enabled.
+    default: enabled
```

Comment:
Default values are dangerous. There should not be a default, and the module should only change the value if the user explicitly specified something.

Reply:
Ok. I'll remove defaults